backend/src/tests/userschema.test.ts

```diff
@@ -0,0 +1,74 @@
+import { createTestClient } from 'apollo-server-testing'
+import gql from 'graphql-tag'
+import { server } from '../index'
+
+const AUTHORIZE_WITH_GH = gql`
+  mutation authorizeWithGithub($code: String!) {
+    authorizeWithGithub(code: $code) {
+      user {
+        id
+        username
+        emails
+        gitHubId
+        gitHubLogin
+        gitHubEmail
+        gitHubReposUrl
+        gitHubToken
+      }
+      token
+    }
+  }
+`
+
+const ME = gql`
+  query {
+    me {
+      id
+      username
+      emails
+      gitHubId
+      gitHubLogin
+      gitHubEmail
+      gitHubReposUrl
+      gitHubToken
+    }
+  }
+`
+
+describe('User schema mutations', () => {
+
+  it('user can not authorize with GitHub without providing a code as parameter', async () => {
+    const { mutate } = createTestClient(server)
+
+    const res = await mutate({
+      mutation: AUTHORIZE_WITH_GH
+    })
+    
+    expect(res.errors?.some(error => error.message === 'Variable "$code" of required type "String!" was not provided.')).toBeTruthy()
```

Comment:
The assert (`expect`) could be easier to read if the response/error is first fetched to a variable and compared after

Reply:
+1

Reply:
Done.